bots/telegram/client.js

```diff
@@ -0,0 +1,150 @@
+const Telegraf = require('telegraf');
+
+const config = require('../../app-config').telegram;
+const bus = require('../../bus');
+const botNetwork = require('./network');
+
+const chat = config[process.env.NODE_ENV];
+
+const {BOT_TOKEN} = config;
+
+const client = new Telegraf(BOT_TOKEN);
+
+class Name {
+  static from(msg) {
+    if (msg.username) {
+      return msg.username;
+    }
+
+    if (msg.last_name) {
+      return `${msg.first_name} ${msg.last_name}`;
+    }
+
+    return msg.first_name;
+  }
+}
```

Comment:
Honestly that looks strange. I can't see why you need a `class` for that single static function.

Reply:
This is done this way for having common interface for all *Message classes, code with similar logic is static and named as test is a part of other messages types. These kinds of methods are for checking if Telegram message could be transformed to our class, we can rename them all or re-structure as non-statics or whatever but in such case this change should be applied to all methods of this kind

Reply:
Ok.